packages/vanilla-bundle/src/components/slick-vanilla-grid-bundle.ts

```diff
@@ -1434,7 +1434,7 @@ export class SlickVanillaGridBundle {
 
     if (this.slickGrid) {
       // find the new column reference pointer & re-assign the new editor to the internalColumnEditor
-      const columns = this.slickGrid.getColumns();
+      const columns = this.columnDefinitions;
```

Comment:
please remove this line and apply it directly below
```diff
- const columns = this.columnDefinitions;
if (Array.isArray(columns)) {
-        const columnRef = columns.find((col: Column) => col.id === column.id);
+        const columnRef = this.columnDefinitions.find((col: Column) => col.id === column.id);
```

Reply:
to remove the "const columns" I also have to change the "if" list. Is it good like this?

![image](https://user-images.githubusercontent.com/44271732/184260581-e7c15e77-dc97-45ae-ba9b-197e97ea837c.png)

Reply:
that seems correct, are you going to be able to fix the unit tests or do you need help?